src/components/Register/Register.js

```diff
@@ -95,7 +97,7 @@ function Register() {
                   Mobile Number
                   <span style={{ color: 'red' }}>*</span>
                 </Form.Label>
-                <Form.Control type="tel" placeholder="" pattern="+[0-9]{3}[0-9]{3}-[0-9]{2}-[0-9]{3}" name="mobileNumber" value={formData.mobileNumber} onChange={handleChange} required className="form-control-custom" />
+                <Form.Control type="tel" placeholder="Enter US phone number" pattern="^(\+0?1\s)?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$" name="mobileNumber" value={formData.mobileNumber} onChange={handleChange} required className="form-control-custom" />
```

Comment:
It is not only US phone number. People can register from India or other countries too.

Reply:
Also, can you add alert/message on required format for phone number and add validation too? I was able to add more than 10